build.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+
+// Run "cargo build -vv" if you want to see gn output.
+// TODO For the time being you must set an env var DENO_BUILD_PATH
+// which might be `pwd`/out/debug or `pwd`/out/release.
+// TODO Currently DENO_BUILD_PATH must be absolute.
+// TODO Combine DENO_BUILD_PATH and OUT_DIR.
+
+use std::env;
+use std::process::Command;
+
+fn main() {
+  let mode = env::var("PROFILE").unwrap();
+  let deno_build_path = env::var("DENO_BUILD_PATH").unwrap();
+
+  let status = Command::new("./tools/setup.py")
```

Comment:
```suggestion
  let status = Command::new("python").arg("./tools/setup.py")
```

Reply:
Also, can we not run setup.py when run in the context of RLS?

Reply:
For the future: I think setup.py does too much to be run as a build step, let's have a separate 'configure.py' or something to run just the `gn gen` step without the download stuff.

Reply:
Fixed. 

> For the future: I think setup.py does too much to be run as a build step, let's have a separate 'configure.py' or something to run just the gn gen step without the download stuff.

Agree that it shouldn't be run on every `cargo build`... I figure it's okay for bootstrapping now.